dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/configuration/JspConfigPropertyGroup.java

```diff
@@ -18,103 +18,16 @@
 
 import com.ibm.ws.javaee.dd.jsp.JSPPropertyGroup;
 
-public class JspConfigPropertyGroup implements JspPropertyGroupDescriptor {
+public interface JspConfigPropertyGroup extends JspPropertyGroupDescriptor {
+   
     /**
-     * Comment for <code>serialVersionUID</code>
+     * This method is under EE10's JspPropertyGroupDescriptor, but we are using the older servlet APIs to build 
+     * this bundle. This method has been added under JspConfigPropertyGroup instead. 
+     * It will only be used when Pages 3.1 is enabled. 
+     * Will the use of an unknown identifier in EL within a JSP page trigger an error for this group?
```

Comment:
Can we avoid the use of "EL / JSP" here in the comments? If specific to EE10  we should be good to use "Jakarta Server Pages page, Expression Language"